packages/cli-kit/src/private/node/ui/components/TextPrompt.tsx

```diff
@@ -44,6 +46,8 @@ const TextPrompt: FunctionComponent<TextPromptProps> = ({
   const {oneThird} = useLayout()
   const [answer, setAnswer] = useState<string>('')
   const answerOrDefault = answer.length > 0 ? answer : defaultValue
+  const useEmptyValue = answerOrDefault === ''
```

Comment:
Could we change this to something like `displayEmptyValue`? Usually things that begin with `use` in react are hooks and it'd be nice to keep that convention.